board-stm32mp157c-dk2.yml

```diff
@@ -0,0 +1,8 @@
+header:
+  version: 8
+
+machine: stm32mp157c-dk2
+
+local_conf_header:
+  psplash: |
+    IMAGE_FEATURES_append = " splash"
```

Comment:
You can drop this part since `splash` is already there in `core-image-base`, see [1].

[1] - https://github.com/openembedded/openembedded-core/blob/master/meta/recipes-core/images/core-image-base.bb#L4